cli/src/main/java/de/jplag/cli/server/RoutingResources.java

```diff
@@ -0,0 +1,31 @@
+package de.jplag.cli.server;
+
+import com.sun.net.httpserver.HttpExchange;
+
+/**
+ * Responds with data from the resources
+ */
+public class RoutingResources implements Routing {
+    private String prefix;
+
+    /**
+     * @param prefix The prefix to use within the resources
+     */
+    public RoutingResources(String prefix) {
+        this.prefix = prefix;
+
+        if (!this.prefix.startsWith("/")) {
```

Comment:
Let us deduplicate all these magic strings via a constant! :D

Reply:
Also, this is system-dependent.

Reply:
Shouldn't be for urls :) or do I miss anything?

Reply:
Ah, right, did not think about what it was being used for here. My bad!